server/app/views/applicant/ApplicantProgramBlockEditTemplate.html

```diff
@@ -0,0 +1,12 @@
+<html>
+    <head th:replace="~{applicant/ApplicantBaseFragment :: pageHeader}"></head>
+    <body>
+        <div>
+            <form id="cf-block-form" th:action="${formAction}" method="POST">
+                <input hidden th:value="${csrfToken}" name="csrfToken">
+                <div th:replace="~{applicant/ApplicantBaseFragment :: requiredFieldsExplanation}"></div>
+            </form>
+        </div>
+        <footer th:replace="~{applicant/ApplicantBaseFragment :: pageFooter}"></footer>
+    </body>
+</html>
```

Comment:
**suggestion**: Can we add a newline at the end of the file?

Reply:
done